dependencies.gradle

```diff
@@ -25,7 +25,7 @@ ext {
   hamcrestIntegration = "org.hamcrest:hamcrest-integration:$hamcrestVersion"
 
   // Other
-  threeTenAbp = 'com.jakewharton.threetenabp:threetenabp:1.1.0'
+  threeTenAbp = 'com.jakewharton.threetenabp:threetenabp:1.1.1'
```

Comment:
This is the heart of this change.  Without the updated version of threetenabp it will fail to compile in later versions of Android Studio 3.3 using Gradle Plugin 3.3.0